src/shared/renderUtils.jsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 
-function renderBody(body){
+function renderBody(body, component){
 	const paragraphs = body.split("\n"); // splits the string at every line-break
 	return paragraphs.map(renderParagraph);
 }// renderBody
 
-function renderParagraph(text, index){
-	const key = `about__body__paragraph_${index}`;
+function renderParagraph(text, index, component){
+	const key = `${component}__body__paragraph_${index}`;
 	return (
 		<p key={ key }>
```

Comment:
```suggestion
		<p className={s.paragraph} key={ key }>
```

Reply:
As render util doesn't have own s.paragraph, now passing the class name in along with the scope.